02-peer-protocol.md

```diff
@@ -142,6 +152,9 @@ allow commitment transactions to propagate through the Bitcoin
 network.  It SHOULD set `htlc_minimum_msat` to the minimum
 amount HTLC it is willing to accept from this peer.
 
+The receiving not MUST failed the channel if `announce_channel` is set
+to `false` (`0x00`), yet they wish to publicly announce the channel.
+
```

Comment:
s/not//node/.  It's a bit vague, though.  I mean, they might wish to announce it, but it might not be a dealbreaker?

How about:

    The receiving node MAY fail the channel if `announce_channel` is `0x00` and it wants to publicly announce the channel.